db/migrations/20181002220302_create_vat_fold.sql

```diff
@@ -4,20 +4,21 @@ CREATE TABLE maker.vat_fold
     id        SERIAL PRIMARY KEY,
     header_id INTEGER NOT NULL REFERENCES headers (id) ON DELETE CASCADE,
     log_id    BIGINT  NOT NULL REFERENCES header_sync_logs (id) ON DELETE CASCADE,
-    urn_id    INTEGER NOT NULL REFERENCES maker.urns (id) ON DELETE CASCADE,
+    ilk_id    INTEGER NOT NULL REFERENCES maker.ilks (id) ON DELETE CASCADE,
+    u         TEXT NOT NULL,
```

Comment:
Should `u` potentially be an int referencing `public.addresses (id)`? Not sure if it matters if this is always just the address for the `vow`, but thinking that would match our current approach elsewhere

Reply:
Didn't see that for `vat_suck` which is very similar, that's why I didn't :)

Reply:
🤔 seems like maybe we should make the change in both places?

Reply:
Hmm maybe that's true.. Would you mind creating tasks for this? I'm quite sure there are more ad-hoc addresses that we don't reference there that we should also catch in that cleanup.